PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py

```diff
@@ -90,6 +90,8 @@
         'keep GenLumiInfoHeader_generator_*_*',
         'keep GenLumiInfoProduct_*_*_*',
         'keep GenEventInfoProduct_generator_*_*',
+        'keep recoGenParticles_genPUProtons_*_*', 
+        'keep recoGenParticles_*_genPUProtons_*',
```

Comment:
@gpetruc, @slava77 
There will be only one collection saved, but there are two possible workflows:
i) Standard mixing, where the collection is saved from the 'genPUProtons' module in the DIGI sequence:
Type                                  Module                      Label             Process   
vector<reco::GenParticle>             "genPUProtons"              ""                "HLT"     

ii) Pre-mixing, where the collection is saved in the premixed sample and later copied by the DataMixingModule ('mixData'), with label 'genPUProtons':
Type                                  Module                      Label             Process   
vector<reco::GenParticle>             "mixData"                   "genPUProtons"    "DIGI2RAW"